kybra/compiler/kybra_generate/src/canister_method/rust.rs

```diff
@@ -31,20 +31,10 @@ impl SourceMapped<&Located<StmtKind>> {
                 let params = tuple::generate_tuple(&param_conversions);
```

Comment:
Look into getting rid of generate_call_to_py_function

Reply:
mmmm. I'm not seeing an easy way to remove this. `generate_call_to_py_function` will take a `Located<StmtKind>` and look at it's name and params and generate all the boilerplate leading up to the `call_global_python_function` method. Some places don't need params so maybe we could just generate a quote that calls `call_global_python_function` but it would still need to do some very similar work to get the name. So, I feel like removing this will just create more inconsistency in the codebase.

I think your thought though was that we could just replace it with `call_global_python_function`, but I don't think we can. This is the boilerplate outside and then `call_global_python_function` does all the internal call and resultify stuff.